helm/compliance-toolkit/keycloak-init-values.yaml

```diff
@@ -11,6 +11,14 @@ keycloak:
           direct_grant_flow_alias: 'direct grant'
           browser_flow_alias: 'browser'
 
+      users:
+        - username: mpartner-default-abis
+          email: abis@gmail.com
+          firstName: mpartner-default-abis
+          lastName: 
+          password: abcd1234
```

Comment:
Please add this in readme document

Reply:
updated the readme @syedsalman3753

Reply:
After logging in, It will ask to change your password hence its temporary one @ckm007